Gemfile.lock

```diff
@@ -51,7 +51,7 @@ GEM
     ast (2.2.0)
     autoprefixer-rails (6.3.6)
       execjs
-    before_commit (0.4.0)
```

Comment:
This change has nothing to do with the rubocop failures. It just reflects the fact that before_commit has been updated.